guides/common/modules/ref_prerequisites-disaster-recovery-by-virtualizing-your-projectserver.adoc

```diff
@@ -0,0 +1,6 @@
+[id="prerequisites-disaster-recovery-by-virtualizing-{project-context}-server"]
+= Prerequisites
+
+// * Review xref:overview-of-recommended-disaster-recovery-plans[] to make sure that this disaster recovery plan works for you.
```

Comment:
I'll uncomment this after/if https://github.com/theforeman/foreman-documentation/pull/3614 gets merged.